src/nt_users_tool/net_commands.py

```diff
@@ -0,0 +1,72 @@
+from os import popen
+from concurrent.futures import Executor, ThreadPoolExecutor
+
+from typing import NamedTuple
+
+from constants import LAST_RELEVANT_ELEMENT_POSITION
```

Comment:
Please use relative imports like this `from .constants ...` since we have a package. Can you please make the change in all imports?

We could also import from the package `from nt_user_tool.constants import ...`

Reply:
Imports are now done with the package syntax nt_user_tool.module.